app/views/partials/_form_profile.html.haml

```diff
@@ -0,0 +1,23 @@
+= form_with model: @profile, local: true do |form|
+  %p
+    = form.label :name
+    %br/
+    = form.text_field :name
+  %p
+    = form.label :location
+    %br/
+    = form.text_field :location
+  %p
+    = form.label :description
+    %br/
+    = form.text_area :description  
+  %p
+    = form.label :price_kr_per_day
```

Comment:
```
= form.label :price, 'Price kr per day'
```

Reply:
we remove the price altogether, that will now be an attribute of Offer model.